packages/common/source/models/MembershipModel.ts

```diff
@@ -0,0 +1,43 @@
+import { Schema, model } from "mongoose";
+
+import { Membership } from "../types";
+import { membershipStatuses, membershipTypes } from "../utils/constants";
+
+const membershipSchema = new Schema(
+    {
+        // An identifier that points to the User whose membership is being defined by the current document.
+        member: {
+            type: Schema.Types.ObjectId,
+            ref: "User",
+            required: true,
+        },
+        // An identifier that points to the User that invited the member to the class specified by division.
+        inviter: {
+            type: Schema.Types.ObjectId,
+            ref: "User",
+            required: true,
+        },
+        // An identifier that points to the division. This attribute is polymorphic, that is, its meaning is defined based on the value of type attribute. For example, if type is organization, then the identifier points to an organization document. On the other hand, if type is group, then the identifier points to a group document.
```

Comment:
Don't write lines longer than 100 characters. IIRC, our workspace is configured to render a margin in VS Code for 100 characters.